File: ProyectoOurTorunament/app/src/main/java/com/example/ourtournament/Objetos/Usuario.java
package com.example.ourtournament.Objetos;

import java.util.Date;

public class Usuario {

    public int IdUsuario;
    public String NombreUsuario;
    public String Contrasenia;
    public Date FechaDeNacimiento;
    public String Email;
    public int GolesEnTorneo;


    public Usuario(int idusuario, String nombreusuario, String contrasenia, Date fechadenacimiento, String email, int golesentorneo)
    {
        IdUsuario = idusuario;
        NombreUsuario = nombreusuario;
        Contrasenia = contrasenia;
        FechaDeNacimiento = fechadenacimiento;
        Email = email;
        GolesEnTorneo = golesentorneo;
    }

    public Usuario()
    {

    }
}

File: OurTournamentAPI/OurTournamentAPI/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class Usuario
    {
        private int _IdUsuario;
        private String _NombreUsuario;
        private String _Contrasenia;
        private DateTime _FechaDeNacimiento;
        private String _Email;
        private int _GolesEnTorneo;

        public int IdUsuario { get => _IdUsuario; set => _IdUsuario = value; }
        public String NombreUsuario { get => _NombreUsuario; set => _NombreUsuario = value; }
        public String Contrasenia { get => _Contrasenia; set => _Contrasenia = value; }
        public DateTime FechaDeNacimiento { get => _FechaDeNacimiento; set => _FechaDeNacimiento = value; }
        public String Email { get => _Email; set => _Email = value; }
        public int GolesEnTorneo { get => _GolesEnTorneo; set => _GolesEnTorneo = value; }

        public Usuario(int IdUsuario, String NombreUsuario, String Contrasenia, DateTime FechaDeNacimiento, String Email, int GolesEnTorneo)
        {
            _IdUsuario = IdUsuario;
            _NombreUsuario = NombreUsuario;
            _Contrasenia = Contrasenia;
            _FechaDeNacimiento = FechaDeNacimiento;
            _Email = Email;
            _GolesEnTorneo = GolesEnTorneo;
        }

        public Usuario()
        {
        }
    }
}